### ISSUE TITLE
NPE Timings when getting block ID

### ISSUE THREAD

**Issue by TomberWolf** [SpongePowered/SpongeForge#1402]
spongeforge-1.10.2-2254-5.2.0-BETA-2259

```
Time: 3/30/17 2:36 PM
Description: Exception ticking world

java.lang.NullPointerException: Exception ticking world
	at net.minecraft.block.Block.getId(Block.java:2623)
	at co.aikar.timings.SpongeTimings.getBlockTiming(SpongeTimings.java:147)
	at net.minecraft.block.Block.getTimingsHandler(Block.java:2798)
	at net.minecraft.world.WorldServer.func_147456_g(WorldServer.java:2091)
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:220)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:698)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471)
	at java.lang.Thread.run(Unknown Source)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at net.minecraft.block.Block.getId(Block.java:2623)
	at co.aikar.timings.SpongeTimings.getBlockTiming(SpongeTimings.java:147)
	at net.minecraft.block.Block.getTimingsHandler(Block.java:2798)
	at net.minecraft.world.WorldServer.func_147456_g(WorldServer.java:2091)
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:220)

```

**Comment by ktole1999** [comment]
need full log. paste link from pastebin

**Comment by simon816** [comment]
From what I can tell it appears that a block is loaded into the world but it not registered in the block registry. What mods do you have?

**Comment by TomberWolf** [comment]
This should be better:

https://pastebin.com/vGYUgVN1

**Comment by sylphio** [comment]
Same bug for us. 

https://pastebin.com/BP9QwCPS

On our side, seem to happen when cutting Fir tree.

**Comment by sylphio** [comment]
Hi, 
  Did a quick debug on my side to identify the improperly registered block.
 (I added a log to detect getId call on block not registered).

For us, it is tile.BOPLeafAir_1  from regrowable Leaf.

Thank

**Comment by Deamon5550** [comment]
Open an issue with the Ice_and_Fire mod, [they aren't registering any of their blocks properly](https://github.com/Alex-the-666/Ice_and_Fire/blob/master/src/main/java/com/github/alexthe666/iceandfire/core/ModBlocks.java#L38-L63) (or at all......)

(At least this is the cause of the most recent ones, others may be other mods, the common thing is that they aren't registering their blocks at all...)

**Comment by kashike** [comment]
@Deamon5550 They are registered:
https://github.com/Alex-the-666/Ice_and_Fire/blob/master/src/main/java/com/github/alexthe666/iceandfire/block/BlockLectern.java#L42
https://github.com/Alex-the-666/Ice_and_Fire/blob/master/src/main/java/com/github/alexthe666/iceandfire/block/BlockDragonOre.java#L24

**Comment by gabizou** [comment]
Closing this issue, as of today 1.10.2 and 1.11.2 are not longer supported. If a sufficiently, decent, and tested Pull Request is made to fix this bug/issue, it will still be considered, however, it is NOT to be expected that any bug fixes made for 1.12.2 will be back ported to 1.11.2 or 1.10.2.

If you can reproduce this issue with SpongeForge on Minecraft 1.12.2 and this issue was wrongly closed, please [open a new issue](https://github.com/SpongePowered/SpongeForge/issues/new) with updated information.

**Comment by Bulldog83** [comment]
I have same crash on 1.12.2 forge server with sponge.
Crash report with list of mods:
https://gist.github.com/Bulldog83/eb800c057029cf01677440c8880e91c4
